portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -180,6 +180,42 @@ definition {
 		}
 	}
 
+	@description = "This test validates Fjord Theme."
```

Comment:
Actually, now that I think about it some more, we don't necessarily a new testcase for this. I think we can add onto current FjordTheme#AddThemeSite test.